readme.md

```diff
@@ -1,77 +1,74 @@
-Stash Build Notifier Plugin for Jenkins
-=======================================
+Bitbucket Server Notifier Plugin for Jenkins
+============================================
 
-This Jenkins plugin notifies Stash of build results. Failed or
-successful builds will show up as little icons in the Stash web 
-interface in commit logs. Clicking on such an icon will take the 
-user to the specific build.
+This Jenkins plugin notifies Bitbucket Server (formerly known as Stash) of build results.
+Failed or successful builds will show up as little icons in Bitbucket's web interface in
+commit logs. Clicking on such an icon will take the user to the specific build.
 
 Requirements
 ============
 
-* **[Stash][] 2.1** or newer. This plugin uses the Atlassian 
-[Stash Build REST API][] which was introduced with Stash 2.1. 
-* **Jenkins 1.498** or newer
+* **[Stash][] 2.1** or newer / **[Bitbucket Server][] 4.0** or newer
+* **Jenkins 1.625.3** or newer
```

Comment:
we actually forgot to mention new requirements here, but this is not a problem of this PR